tests/phases/test_contents.py

```diff
@@ -19,3 +20,16 @@ class DummyPhase(PhaseContent):
 
     assert phase.identifier == 'dummy:999:phase'
     assert str(phase) == 'DummyPhase (dummy:phase)'
+
+
+def test_phase_features():
+    class DummyPhase(PhaseContent):
+        app = 'dummy'
+        phase = 'phase'
+        weight = 9999
+        features = {
+            'comment': (models.Post,)
+        }
```

Comment:
Because you are reusing the dummy from `test_phase_content()` I would suggest to add a factory or at least a fixture to create the dummy.